install/kubernetes/addons/grafana.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: grafana
+  namespace: istio-system
```

Comment:
i don't know that it really matters, but should we (if it is possible) have the addons be in a separate namespace?

Reply:
it is possible, but it is an enhancement.